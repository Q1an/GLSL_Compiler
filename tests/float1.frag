float
float x;
float y = 1.0;
