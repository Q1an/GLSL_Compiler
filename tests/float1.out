float        line 1 cols 1-5 is T_Float 
float        line 2 cols 1-5 is T_Float 
x            line 2 cols 7-7 is T_Identifier 
;            line 2 cols 8-8 is ';' 
float        line 3 cols 1-5 is T_Float 
y            line 3 cols 7-7 is T_Identifier 
=            line 3 cols 9-9 is '=' 
1.0          line 3 cols 11-13 is T_FloatConstant (value = 1)
;            line 3 cols 14-14 is ';' 
